fake.py

```diff
@@ -297,10 +313,11 @@ def compose_frame(self, frame):
         if self.no_background is False:
             background_frame = next(self.images["background"])
         else:
-            background_frame = cv2.blur(frame,
-                                        (self.background_blur,
-                                         self.background_blur),
-                                        cv2.BORDER_DEFAULT)
+            background_frame = cv2.GaussianBlur(frame,
+                                                (self.background_blur,
+                                                 self.background_blur),
+                                                self.sigma,
+                                                borderType=cv2.BORDER_DEFAULT)
```

Comment:
Sorry, my refactor on save was on. This was the blur change

Reply:
Well, this script needed linting a long time ago. :P

Reply:
Which editor do you use, btw?

Reply:
I use emacs with elpy, you?

Reply:
I wrote this mainly in either Kate or Kwrite. Recently I have been using Visual Studio Code.

Reply:
Yeah nice, I've heard good things about Visual Studio Code.